.github/workflows/release.yml

```diff
@@ -129,6 +129,26 @@ jobs:
       DOCKER_USERNAME: ${{ secrets.DOCKER_USERNAME }}
       DOCKER_TOKEN: ${{ secrets.DOCKER_TOKEN }}
 
+  keycloak-operator-hub-publish:
```

Comment:
I've also updated the release.yml. If necessary, it can be commented on for now.